flow/core/kernel/pedestrian/util.py

```diff
@@ -0,0 +1,4 @@
+import math
+
+def orientation_unit_circle(angle):
+    return (360 - (angle - 90)) % 360
```

Comment:
Please leave a comment explaining the way SUMO puts the angle so it's clear why you do it this way